crates/gpui/src/text_system.rs

```diff
@@ -89,6 +89,7 @@ impl TextSystem {
                 .iter()
                 .map(|font| font.family.to_string()),
         );
+        names.insert(".SystemUIFont".into());
```

Comment:
I think it's better to add this line of code because the `settings.json` mentions that one can set the font to `.SystemUIFont`, but the JSON schema does not include this font.

Should I open a new PR for this?

Reply:
Yep, a new PR would be great!